test/client/index_test.rb

```diff
@@ -367,11 +372,12 @@
 
       @index.percolator("1").create :query => { :match_all => { } }
       @index.percolator("2").create :query => { :match => { :author => "pea53" } }
+      @index.refresh
 
       h = @index.multi_percolate(:type => "doco") do |m|
         m.percolate :author => "pea53"
         m.percolate :author => "grantr"
-        m.count({}, { :author => "grantr" })
+        m.count({ :author => "grantr" }, {})
```

Comment:
What's the deal with flipping these hashes in the params list? Does it represent an API change?

Reply:
Yeah this was a wierd one!

After adding the param merges I noticed in `byebug` that this `count` call was generating a bad output (injecting the `doc` into the params/`percolate` bulk line and a blank doc into the for the `count`)

The API docs in the [code](https://github.com/github/elastomer-client/blob/master/lib/elastomer/client/index.rb#L496) indicate these were backwards and should match the format of the `percolate` (in which the 2nd arg params hash is optional).

I see no uses of `multi_percolate` in `github/github` so I'm guessing this just silently passed it's test and went unused all this time? Line Git-blames to 2015, so a bit scary, but the output looks right now, and is [wrong on master](https://gist.github.com/2909d8aa40479a7cf0b424bb9304a0b2) so little creepy to flip it I agree, but seems legit? Happy to investigate further if anyone suspects I got turned around here and am being too brave :)

Reply:
Whoa crazy! Unfixed bug from way back. Since we're going to bump the major version anyway, I think a breaking change is OK. Can you add it to the CHANGELOG as part of this PR so we don't forget about it?

Reply:
Good call, will update now!